lib/kotkowo_web/controllers/error/404.html.heex

```diff
@@ -0,0 +1,17 @@
+<.section class="mt-8  flex font-manrope lg:flex-row flex-col justify-center items-center gap-x-10">
+  <img src={~p"/images/404_cat.svg"} width="266" height="353" class="lg:trasnlate-x-0 translate-x-4"/>
+  <div class="flex flex-col gap-y-6 lg:text-justify text-center mx-6 lg:mx-0 mt-10 lg:mt-0">
+    <h1 class="text-4xl font-extrabold">Ups, kotki znowu coś spsociły...</h1>
+
+    <h2 class="font-bold text-justify tracking-tight">Strona nie istnieje. Oto, co możesz zrobić:</h2>
+    <ol class="font-inter text-justify">
+    <li>1. Na poprawę humoru <a href="#" class="underline font-bold text-primary">obejrzyj kotki do adopcji z Fundacji Kotkowo.</a></li>
+    <li>2. Przybij łapkę i <a href="#" class="underline font-bold text-primary">dorzuć się do smaczka!</a></li>
+    </ol>
```

Comment:
Try `list-decimal` instead of manually using `1., 2. ...`.
https://tailwindcss.com/docs/list-style-type